projects/power_distribution/src/main.c

```diff
@@ -158,6 +159,9 @@ int main(void) {
       .strobe_blink_delay_us = STROBE_BLINK_INTERVAL_US,
     };
     rear_power_distribution_strobe_blinker_init(&strobe_blinker_settings);
+  } else {
+    // initialize UV cutoff detector
+    front_uv_detector_init(&(GpioAddress)FRONT_UV_COMPARATOR_PIN);
   }
```

Comment:
Having this as the `else` clause creates a kind of confusing if statement like this:
```c
if (!s_is_front_power_distribution) {
  // ... initialize strobe blinker for rear
} else {
  // ... initialize UV cutoff detector for front
}
```
It's a bit ugly since the `else` clause is based on a double negative. I'd prefer you refactor it to look like this:
```c
if (s_is_front_power_distribution) {
  // ... initialize UV cutoff detector for front
} else {
  // ... initialize strobe blinker for rear
}
```
That way you get rid of the double negative. When you do, move the comment about the rear strobe blinker to inside the else clause so it stays with that block of code.